Formula/a/ansible-lint.rb

```diff
@@ -99,8 +98,8 @@ class AnsibleLint < Formula
   end
 
   resource "wcmatch" do
-    url "https://files.pythonhosted.org/packages/92/51/72ce10501dbfe508808fd6a637d0a35d1b723a5e8c470f3d6e9458a4f415/wcmatch-8.5.tar.gz"
-    sha256 "86c17572d0f75cbf3bcb1a18f3bf2f9e72b39a9c08c9b4a74e991e1882a8efb3"
+    url "https://files.pythonhosted.org/packages/38/c6/0c5f324561c9396868d6badf571590c1a7802a81180c3097e4dfdc2f35c0/wcmatch-8.5.1.tar.gz"
+    sha256 "c0088c7f6426cf6bf27e530e2b7b734031905f7e490475fd83c7c5008ab581b3"
   end
 
   def install
```

Comment:
that should be only for sonoma builds, right?

cc @Homebrew/core for thoughts

Reply:
That'd be correct, yeah. Since Ventura doesn't support Xcode 15.3. So maybe this is more appropriate:

```ruby
ENV.append_to_cflags "-Wno-incompatible-function-pointer-types" unless DevelopmentTools.clang_build_version >= 1500
```

Reply:
Based on https://github.com/llvm/llvm-project/commit/af01f717c48f0fd2481600ed6c00441763365b62, I think it is fine to temporarily disable error until `ruamel` fix.

Reply:
Exact conditional is tricky as `clang_build_version` didn't change in 15.3, but 1500 is OK if paired with a comment that explains this only applies to Xcode 15.3+ and until upstream fixes the bug. The comment in your suggestion is good, so LGTM.